assets/js/123validations.js

```diff
@@ -0,0 +1,62 @@
+function content_for_stepIsValid(clickedButton){
```

Comment:
Identifier 'content_for_stepIsValid' is not in camel case.<br>'content_for_stepIsValid' is defined but never used.